### ISSUE TITLE
[1.16.4] holding CTRL does not display unused keys

### ISSUE THREAD

**Issue by mindsculptorjace** [jaredlll08/Controlling#76]
Not sure if this is a bug or just not implemented yet, thought it was worth bringing to your attention at least

**Comment by jaredlll08** [comment]
Feature was removed because I don't have a good way to show it that fits with the vanilla style.

**Comment by Janitor53** [comment]
Dude, it was a most useful feature for the large modpacks! Can you bring it back please?

**Comment by stickybeard** [comment]
I would also like to vote in favor of this feature.  Could it maybe be enabled/disabled via server-side config?

**Comment by jaredlll08** [comment]
Will re-look into it then.

> Could it maybe be enabled/disabled via server-side config?

Did you mean to say a client side config? This is a client side only mod so the server does not and should not have control over the client's settings.

**Comment by stickybeard** [comment]
> Did you mean to say a client side config? This is a client side only mod so the server does not and should not have control over the client's settings.

My apologies, for some reason I got it in my head you said it was removed for admin reasons.  I said server-side to accommodate that misperception.   Now that I have reread you initial reason, a client side config makes much more sense.  I am a recent addition to the modded MC community, so I have not have a chance to utilize the unused key feature previously.   I am apparently somewhat of a modaholic, so this would make my game a lot more accomodating.  Even if it does not come to fruition, thank you for the consideration.
